_posts/2024-06-14-this-week-in-rails.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "New transaction event, bugfixes and more!"
+categories: news
+author: Greg
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2024-06-14
+---
+
+
+Hi, it's [Greg](https://greg.molnar.io). Let's explore this week's changes in the Rails codebase.
+
+[Only sanitize _content_ attribute when present in attachments](https://github.com/rails/rails/pull/52093)  
+As a recent security fix introduced sanitizing the `content` attribute
+of `ActionText::Attachable::ContentAttachment` and as a side effect, it always
+sets the attribute, and when attribute is set for Action Text attachments, Trix shows this
+content instead of the image preview. This pull request fixes that issue. 

```

Comment:
```suggestion
[Only sanitize _content_ attribute when present in attachments](https://github.com/rails/rails/pull/52093)  
A recent security fix introduced sanitizing the `content` attribute
of `ActionText::Attachable::ContentAttachment`. As a side effect, it always
sets the attribute, and when the attribute is set for Action Text attachments, Trix shows this
content instead of the image preview. This pull request fixes that issue. 
```

Or maybe even:
```suggestion
[Only sanitize _content_ attribute when present in attachments](https://github.com/rails/rails/pull/52093)  
A recent security fix introduced sanitizing the `content` attribute
of `ActionText::Attachable::ContentAttachment`. As a side effect it always
sets the attribute, even when it's missing, causing Trix to not show
the image preview. This pull request fixes that issue. 

```